### PR TITLE
chore: update CODEOWNERS to team that uses these tests

### PR DESCRIPTION
**Pull request by codyoss**

### CODE REVIEW

**Comment by codyoss** [comment]
cc @crwilcox Are you good with these changes?

**Review by JustinBeckwith — APPROVED** [review]
LGTM, but please let @danielduhh have a look too.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,5 +4,5 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-*     @googleapis/yoshi
+*     @googleapis/api-storage
```
Comment (BenWhitehead) [inline .github/CODEOWNERS:7]:
```suggestion
*     @googleapis/api-storage @googleapis/api-firestore @googleapis/api-bigtable 
```

**Review by BenWhitehead — COMMENTED** [review]
Storage, Firestore and BigTable all use this repo ensuring all of those teams are able to review and merge.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -4,5 +4,4 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-*     @googleapis/yoshi
-
+*     @googleapis/api-storage @googleapis/api-firestore @googleapis/api-bigtable
```
Comment (danielduhh) [inline .github/CODEOWNERS:7]: storage dpe team recently moved to the `cloud-storage-dpes` group

**Pull request merged**